Remove unused file after migrating to Debian Buster Image

championshuttler opened the pull request:
Follow up on #13985 

This file is not needed anymore I think

diox commented:
So, it's currently used in the `Dockerfile` and `Dockerfile.deploy`, there is a `ADD docker/debian-buster-repo /etc/apt/sources.list.d/buster.list` line. 

Do check in the container, but I think it's causing a duplicate apt source and should indeed be removed. In that case the `ADD` line in the `Dockerfile` and `Dockerfile.deploy` should go away as well.

championshuttler commented:
> So, it's currently used in the `Dockerfile` and `Dockerfile.deploy`, there is a `ADD docker/debian-buster-repo /etc/apt/sources.list.d/buster.list` line.
> 
> Do check in the container, but I think it's causing a duplicate apt source and should indeed be removed. In that case the `ADD` line in the `Dockerfile` and `Dockerfile.deploy` should go away as well.

@diox I removed that line in `Dockerfile` last patch but `accidentally ` left in `Dockerfile.deploy` . Removed that now :)

championshuttler commented:
One more question, why there is a merge commit generated for my last patch https://github.com/mozilla/addons-server/commit/97f0aef666d660ac61c957e3671e4d12a1b209dc
